### PR TITLE
export validation

### PR DESCRIPTION
**Pull request by dczulada**
Pull requests into Cypress require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/CYPRESS-838
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

**Comment by dczulada** [comment]
Example report here

[newnew_6053ac10c1c38832fde56d40_report (65).zip](https://github.com/projecttacoma/cvu-fhir/files/6173777/newnew_6053ac10c1c38832fde56d40_report.65.zip)

**Comment by dczulada** [comment]
[new_product_report.zip](https://github.com/projecttacoma/cvu-fhir/files/6185045/new_product_report.zip)

Screen Shot 2021-03-22 at 4 13 07 PM

File: lib/cypress/create_download_zip.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+module Cypress
+  class CreateDownloadZip
+    def self.create_validation_report_zip(_vendor, options)
+      file = Tempfile.new("combined-report-#{Time.now.to_i}")
+      Zip::ZipOutputStream.open(file.path) do |z|
+        add_file_to_zip(z, 'product_report.html', options[:report_content])
+
+        # TODO: Do we do anything by patient?
```
Comment (lmd59) [inline lib/cypress/create_download_zip.rb:8]: Okay with getting rid of this for now then?

**Review by lmd59 — APPROVED** [review]
